src/config.js

```diff
@@ -0,0 +1,11 @@
+// Load different configs for production or development
+
+let configFile = "dev.js";
+
+if (process.env.NODE_ENV === "production") {
+    configFile = "prod.js";
+}
```

Comment:
Yes. We'll set NODE_ENV off ENVIRONMENT inside the Docker entrypoint (`invoke`) script.